### ISSUE TITLE
It's possible to make the tabs draggable inside the same docking panel?

### ISSUE THREAD

**Issue by mmasdivins** [hlorenzi/react-dockable#1]
Hello,
First of all I want to say that your package is amazing. I was wondering if it was possible to make the tabs that are on the same docking panel could be sorted dragging them horizontally? Like the tabs of a web browser.
Thanks for your attention!

**Comment by hlorenzi** [comment]
Yes, that's a feature I want to add! Out of curiosity, have you already imported this library in your own project, or did you just try out the example?

**Comment by mmasdivins** [comment]
Nice! I've downloaded the sample and try out, I really liked and I thought it would be cool if the tabs could be sorted dragging them. Also, it would be amazing if the tabs could communicate in some way between them.

**Comment by mmasdivins** [comment]
Hello, 
I was looking again this poject and I was wondering if there was any news on this feature?
Thanks

**Comment by hlorenzi** [comment]
Sorry, @mmasdivins, I have been focusing on other projects recently. I don't know if I'll get around to doing it anytime soon, unfortunately.

**Comment by mmasdivins** [comment]
Don't worry and thanks for this awesome project!

Missatge de Lorenzi ***@***.***> del dia dt., 15 de nov. 2022
a les 22:07:
